### PR TITLE
[SERV-396] Setup GitHub Actions

### PR DESCRIPTION
**Pull request by markmatney**
Should this repo also have a `release-drafter.yml` and `release.yml` a la Hauth)?

### CODE REVIEW

**Comment by markmatney** [comment]
Nevermind, answered my own question by looking at the ticket. Still in progress.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  pull_request:
+    branches:
+      - main
+jobs:
+  build:
+    name: Maven PR Builder (JDK ${{ matrix.java }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 11 ]
```
Comment (ksclarke) [inline .github/workflows/build.yml:12]: Is there a reason to not do 17 here too?

Comment (markmatney) [reply .github/workflows/build.yml:12]: Tried it on an earlier run but it failed: https://github.com/UCLALibrary/libcal-proxy/runs/6517168845?check_suite_focus=true#step:4:113

Comment (markmatney) [reply .github/workflows/build.yml:12]: And since we're deploying with Java 11 anyway I guess I don't understand why we should build with 17

**Review by ksclarke — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (ksclarke) [reply .github/workflows/build.yml:12]: I'd like us to start moving projects towards 17 since that's the current LTS, but that shouldn't hold us up.

Comment (markmatney) [reply .github/workflows/build.yml:12]: I am in favor of using 17 for this project, fwiw

**Review by DRickard — APPROVED** [review]

**Pull request merged**